notebooks/stage/raw_to_meta.ipynb

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
<p>No pushing to Ceph is happening here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing_list_analysis_toolkit/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>